backend/routes/passRoutes.js

```diff
@@ -1,11 +1,17 @@
 const express = require("express")
 const router = express.Router()
-const { protect } = require("../middleware/authMiddleware")
+const { protect, authorize } = require("../middleware/authMiddleware")
 
-const { generatePass, viewPasses } = require("../controllers/passController")
+const {
+  generatePass,
+  viewPasses,
+  verifyPass,
+} = require("../controllers/passController")
 
-router.post("/generatePass", protect, generatePass)
+router.post("/generatePass", protect, authorize("admin"), generatePass)
 
-router.get("/viewPasses", protect, viewPasses)
+router.get("/viewPasses", protect, authorize("admin"), viewPasses)
+
+router.put("/verifyPass/:_id", protect, authorize("gaurd"), verifyPass)
```

Comment:
Typo of guard.

Reply:
sorry, by mistake, corrected now

Reply:
Can you suggest something better than using strings (guard, admin) everywhere. This type of typo can easily be done but tough to find.

Reply:
ummm...it was storing strings only, otherwise throwing don't know what what errors...just did a hit and trial at this method as well in the routes, so not sure that anything else will work

Reply:
[Ca](https://www.udacity.com/blog/2021/08/developers-guide-to-cpp-constant-variables.html#:~:text=From%20a%20programming%20standpoint%2C%20a,ultimately%20a%20more%20streamlined%20approach.)

`From a programming standpoint, a constant variable tells anyone looking at the code that it contains values that should not be tampered with. This makes for clearer written code and ultimately a more streamlined approach.`

Now use what comes into your mind.